test-cases/longevity/custom_d1.yaml

```diff
@@ -0,0 +1,31 @@
+test_duration: 1500
+n_db_nodes: 6
+n_loaders: 2
+n_monitor_nodes: 1
+
+prepare_write_cmd: [
+  "cassandra-stress user profile=scylla-qa-internal/custom_d1/posts_dataset.yaml cl=QUORUM  'ops(insert=1)' n=5M -rate 'threads=200' -pop 'seq=0..5M'",
+  "cassandra-stress user profile=scylla-qa-internal/custom_d1/posts_dataset.yaml cl=QUORUM  'ops(insert=1)' n=5M -rate 'threads=200' -pop 'seq=5M..10M'"
+                      ]
+stress_cmd: "cassandra-stress user profile=scylla-qa-internal/custom_d1/posts_dataset.yaml cl=QUORUM  'ops(insert=5,update1=3,read1=50,delete1=1)' duration=720m -pop 'dist=gauss(0..10M)'"
+gce_instance_type_db: 'n2-highmem-32'
+gce_instance_type_loader: 'e2-standard-4'
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_seed: '112'
+nemesis_interval: 5
+
+user_prefix: 'lukasz-3h-custom-d1'
+
+gce_n_local_ssd_disk_db: 6
```

Comment:
see: https://cloud.google.com/compute/docs/disks/local-ssd#:~:text=Each%20Local%20SSD%20disk%20is,that%20hosts%20your%20VM%20instance.

I don't think 6 would work